dispatch/dispatch.go

```diff
@@ -204,7 +205,7 @@ type aggrGroup struct {
 	timeout func(time.Duration) time.Duration
 
 	mtx        sync.RWMutex
-	alerts     map[model.Fingerprint]*types.Alert
+	alerts     *store.Alerts
```

Comment:
if you feel it is necessary, you could move `alerts *store.Alerts` above as it isn't guarded by `mtx` anymore.

Reply:
that's a good point, thanks for noticing that!